io.catenax.single_level_bom_as_specified/2.0.0/SingleLevelBomAsSpecified.ttl

```diff
@@ -0,0 +1,211 @@
+#######################################################################
+# Copyright (c) 2022 T-Systems International GmbH
```

Comment:
adjust date to 2022, 2023 for all partners who have contributed to the latest version of the model